data/publiccloud/ssh_config

```diff
@@ -2,6 +2,6 @@ ControlMaster auto
 ControlPath /tmp/ssh_%r_%h_%p
 StrictHostKeyChecking no
 HostKeyAlgorithms ssh-rsa
-IdentityFile ~/.ssh/id_rsa
+IdentityFile SSH_KEY
```

Comment:
```suggestion
IdentityFile $SSH_KEY
```
Since this is a "variable" that needs to be replaced, I would mark it better. Sure the all-caps help but a dollar sign might make it even more obvious.

What do you think?

Reply:
I agree, other tests are using `%SSH_KEY%`